3.9/debian-10/rootfs/opt/bitnami/scripts/librabbitmq.sh

```diff
@@ -755,7 +755,8 @@ rabbitmq_initialize() {
         am_i_root && chown -R "$RABBITMQ_DAEMON_USER:$RABBITMQ_DAEMON_GROUP" "$dir"
     done
 
-    if ! is_mounted_dir_empty "$RABBITMQ_DATA_DIR"; then
+    # use realpath to avoid symlink issues
```

Comment:
Could you please reference this PR in the comment? This way others will know why this was needed.
```suggestion
    # Use realpath to avoid symlink issues
    # ref: https://github.com/bitnami/bitnami-docker-rabbitmq/pull/184
```

Reply:
Thanks, I applied your suggestion.